editor/script/script_text_editor.cpp

```diff
@@ -3009,6 +3062,15 @@ ScriptTextEditor::ScriptTextEditor() {
 	errors_panel->set_focus_mode(FOCUS_CLICK);
 	errors_panel->hide();
 
+	drag_info_label = memnew(Label);
+	drag_info_label->set_anchors_preset(Control::PRESET_BOTTOM_RIGHT);
+	drag_info_label->set_focus_mode(FOCUS_ACCESSIBILITY);
+	drag_info_label->add_theme_color_override(SceneStringName(font_color), Color(0.6f, 0.6f, 0.6f, 1));
+	drag_info_label->add_theme_color_override("font_shadow_color", Color(0.2f, 0.2f, 0.2f, 1));
+	drag_info_label->add_theme_constant_override("shadow_outline_size", 1 * EDSCALE);
+	drag_info_label->add_theme_constant_override("line_spacing", 0);
+	code_editor->get_text_editor()->connect("mouse_exited", callable_mp(this, &ScriptTextEditor::_on_mouse_exited));
```

Comment:
Would the `MOUSE_EXITED` notification also work? since the code_edit is a child.